server/models/express-usage-record.js

```diff
@@ -0,0 +1,38 @@
+var debug = require('debug')('strong-mesh-models:server:express-usage');
+var deleteOld = require('./delete-old');
+
+module.exports = function(ExpressUsageRecord) {
+  function recordUsage(instanceId, req, callback) {
+    debug('Express usage record: %j', req.record);
+    var ServiceProcess = ExpressUsageRecord.app.models.ServiceProcess;
+    var record = req.record;
+    var pid = +record.process.pid;
+
+    var q = {where: {
+      serviceInstanceId: instanceId,
+      pid: pid,
+      stopTime: null
+    }};
+
+    ServiceProcess.findOne(q, function(err, proc) {
+      if (err) return callback(err);
+      if (!proc) {
+        debug('Ignoring metrics for unknown worker pid: %d', pid);
+        return deleteOld();
```

Comment:
Shouldn't `callback` be passed? Otherwise there appears to be code paths where `recordUsage()` never "returns" (in the async sense).